### ISSUE TITLE
Power Delivery options.

### ISSUE THREAD

**Issue by landonracer109** [xJon/Tekkit-2#184]
With the Buff that ICC Reactors got on their power outputs the maximum I can make out of a reactor now is 50k eu per tick. With that I HAVE to have 3-4 adjustable transformers to handle that much throughout on each reactor. Is there a adjustment or something we can do that can handle more throughout of eu? I mean I rlly feel like if the power can be made there should be a machine with capacity to take the eu in directly, or atleast on with 1 transformer.

**Comment by xJon** [comment]
PESUs and Plasma Cables can accept and transfer respectively a voltage of up to 32,768, which is useful. You can also check out the [Battery Station](https://github.com/TinyModularThings/IC2Classic/wiki/Storage-Units).

**Comment by landonracer109** [comment]
I understand the pesu and palsma cables, but if I try to use those they blow up. I have to have 3/4 separate adjustable transformers to not have it all blow up.

**Comment by xJon** [comment]
Hmm what do you think @Speiger?

**Comment by landonracer109** [comment]
I mean if I’m missing something please inform lol, if you’d like an example I can send an image

**Comment by xJon** [comment]
Sending a screenshot would probably be useful :)

**Comment by landonracer109** [comment]
For sure. Give me a few minutes to get to the computer and such.

**Comment by landonracer109** [comment]
![Uploading 03F4E2B1-9D91-42A1-AD7E-1CE3B376A313.jpeg…]() 
Then there’s another on the back you can’t see. But that’s the only way I can get the power out of my reactors.

**Comment by landonracer109** [comment]
![7B88D196-A1F6-4C65-A554-6F97440BB4B4](https://user-images.githubusercontent.com/132149102/235310150-341c261d-0e76-4a0f-aae1-f01f4158a77a.jpeg)

**Comment by landonracer109** [comment]
You. Can def see it now I think it didn’t finish uploading.

**Comment by Speiger** [comment]
@xJon @landonracer109 yeah I am fully aware of that.
But i have no desire to add support for higher tier EU for 1.12.2 or 1.19.2 for that matter.
It is a "Limitation" you need to deal with.
Maybe instead of making the "Output" higher maybe design a way to make your input being used more efficiently.
And yes i am pretty sure that one is a Uranium Waster xD

But back to topic.
32kEU is your Limit and that is what you need to plan around.

**Comment by landonracer109** [comment]
I don’t know if I’m allowed to @ you but I’m replying to you so I will sorry in advance. @Speiger i get where you’re coming from but upgrades to any machine to take a higher tic when the cables won’t transfer it anyways doesn’t make sense. And side note yes it uses tons of uranium. And coolant cells. But it works and I produce the enriched uranium fast enough. But back on topic, there’s definitely reasons for the eu tho, main reason being for UU-Matter. I simply can’t make enough to run factory’s bc so many high end crafts need plasma cores. Lemme rephrase I CAN but it’s very very janky and the I’ve found I gotta make about a million EU per tick to be where I wanna be haha. But. Thank you.

**Comment by Speiger** [comment]
@landonracer109 you can ping me.

About why not allow cables but machines: Because machines just allow stacksize and just simply add to the existing tier.
I mean if you want i can add a block that stops it at tier 6 and you can no longer feed it higher tier stuff even if a machine is required to have 512kEU/t for a recipe (which is possible). 

The whole point is: The Upgrade system allows higher tiers because it just came simplified with it no specific design that had to be made.

The cables stop at LuV because that is how far the mod was designed in the progression.
If I want to add higher tiers i have to add progression for those tier tooo. That includes transformers.
in other words: You ask for more content to be added.

Also cables can transfer infinite amount of EU per tick its just that each packet isn't allowed to be higher then x amount of EU.

If you want higher tier support, ask a addon dev to add it, or add it yourself :)
Because LuV is the tier ic2c stops at :)

Not much arguing here its just a fact that will not change. The mod is AS IS (-bugs, and i define what is a bug)

**Comment by landonracer109** [comment]
I understand. Thank you!
